lib/src/package.dart

```diff
@@ -110,6 +110,14 @@ class Package {
         orElse: () => null);
   }
 
+  /// Returns the path to the pubspec file at the root of the entrypoint, or
+  /// null if no pubspec file is found.
+  String get pubspecPath {
```

Comment:
I don't think this belongs here (Also I don't think you are actually calling this).

`Entrypoint.pubspecPath` should be used for this.